### PR TITLE
introduce ManagementLink with some infos on CasC and reload button

### LINKED ISSUE #6: [Feature] Redeployment after yaml updated
When we update the global configuration in jenkins.yml in the git repository, we need the jenkins master to refresh the configuration without restarting.

### PR DESCRIPTION
**Pull request by ndeloof**
see https://github.com/jenkinsci/configuration-as-code-plugin/issues/6

### CODE REVIEW

**Comment by ndeloof** [comment]
capture d ecran 2018-02-05 a 16 24 39

**Comment by ndeloof** [comment]
not sure about the best way to format Date for jenkins UI, with user's local support

**Comment by daniel-beck** [comment]
@ndeloof The footer in Jenkins uses `xmlns:i="jelly:fmt"` and `<i:formatDate value="…" type="both" dateStyle="medium" timeStyle="long"/>`, as does the build history.

File: src/main/java/org/jenkinsci/plugins/casc/BaseConfigurator.java [hunk 1]
```diff
@@ -157,15 +157,18 @@ protected void configure(Map config, T instance) throws Exception {
             final String name = attribute.getName();
             final Object sub = removeIgnoreCase(config, name);
             if (sub != null) {
+                final Class k = attribute.getType();
+                final Configurator configurator = Configurator.lookup(k);
+                if (configurator == null) throw new IllegalStateException("No configurator implementation to manage "+k);
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/plugins/casc/BaseConfigurator.java]: Throw IOException or another checked one in all entries?

**Comment by oleg-nenashev** [comment]
@ndeloof I think we will need to work on annotations and stream handling before the release. LGTM for now

**Comment by ewelinawilkosz** [comment]
it works!
the problem is after I hit 'Reload' button at http://localhost:8080/jenkins/configuration-as-code page it tries to take me to http://localhost:8080/jenkins/configuration-as-code/configuration-as-code which doesn't exists. The reconfiguration happens though

**Comment by ewelinawilkosz** [comment]
super nice!

**Pull request merged**